database/seeds/test_data/12_test_visit_demo_data.seed.js

```diff
@@ -201,6 +201,242 @@ exports.seed = async (client) => {
         ethnicity_id: 1,
         disability_id: 1,
       },
+
+
+      /*
+       * Users intended as visitors to demonstrate various conflicts in the "Sign in with name" feature
+       */
+
+      // 01. name -> postCode -> GO  (Dupe name, unique postCode)
+      //
+      // 01 (foo,  SH3 4ND,  null,      null,        foo1@example.com),
+      //    (foo,  BR8 9EE,  null,      null,        foo2@example.com),
+      {
+        user_name: 'foo',
```

Comment:
Can we get more usable names for this test data. I can imagine Edward using these to demo the app to new subscribers.

Something like 'visitor a' ect.

Reply:
Happy to make the names easier to remember, but I suspect having super-fake values for post code/phone number might eventually come back to bite us if we add stricter validation on those fields.